attributes/default.rb

```diff
@@ -21,6 +21,7 @@
 default['strongdm']['admin_token'] = nil
 
 # Installation user
+default['strongdm']['url'] = 'https://app.strongdm.com/releases/cli/linux'
```

Comment:
I would probably call it `download_url` just to prevent future ambiguity.

Reply:
This tends to be pretty common in Chef cookbooks, but I can see where it may make a difference. I'm going to just leave it, as is, for now. I have some other things in-flight and I don't want to rebase everything under the sun. :grin: